src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -7316,6 +7316,15 @@ components:
               type: number
       description: Range format program data, part of a CriteriaGroup.
 
+    SnapshotBuilderDomainCriteria:
+      allOf:
+        - $ref: '#/components/schemas/SnapshotBuilderCriteria'
+        - type: object
+          properties:
+            domain:
```

Comment:
This makes sense, but maybe it should be `domain_id`, since it's not an entire `domain` object?

Reply:
It's not the domain ID? I was thinking it would be the concept ID for the root concept for that domain. But I guess a name would be OK to here too; the service will need to know the domain's name and root concept ID so either would be searchable.

Reply:
I _think_ we've been returning the string names, but we can verify that - not too tricky to change it either way if it is wrong.

Reply:
Ah, the `DomainOptions` have both:
```
new SnapshotBuilderDomainOption()
                      .id(12)
                      .category("Observation")
                      .root(
                          new SnapshotBuilderConcept()
                              .id(300)
                              .name("Observation")
                              .count(100)
                              .hasChildren(true))
```

I can PR a change to move to IDs if we'd like, it should be straightforward enough. That said, I think domain name is unique, and looking at the `domain_id` field in https://console.cloud.google.com/bigquery?project=broad-dsde-staging&ws=!1m5!1m4!4m3!1sbigquery-public-data!2scms_synthetic_patient_data_omop!3sconcept, it seems to use the names rather than the IDs, so it may add extra steps to use the ID